suite-native/atoms/src/Button/Button.tsx

```diff
@@ -297,25 +300,33 @@ export const Button = ({
                     style,
                 ]}
             >
-                <HStack alignItems="center">
-                    {viewLeft && (
-                        <ButtonAccessoryView
-                            element={viewLeft}
-                            iconColor={iconColor}
-                            iconSize={size}
-                        />
-                    )}
-                    <Text textAlign="center" variant={buttonToTextSizeMap[size]} color={textColor}>
-                        {children}
-                    </Text>
-                    {viewRight && (
-                        <ButtonAccessoryView
-                            element={viewRight}
-                            iconColor={iconColor}
-                            iconSize={size}
-                        />
-                    )}
-                </HStack>
+                {isLoading ? (
+                    <Loader color={textColor} />
```

Comment:
nit: just thinking what some little fade in animation or something? would that look good?

Reply:
fixed (already squashed)